app/src/main/java/com/onegini/mobile/exampleapp/OneginiSDK.java

```diff
@@ -43,7 +43,7 @@
 public class OneginiSDK {
 
   public static OneginiClient getOneginiClient(final Context context) {
-    OneginiClient oneginiClient = OneginiClient.getInstance();
+    OneginiClient oneginiClient = OneginiClient.Companion.getInstance();
```

Comment:
After adding @JvmStatic annotation in the SDK calling `.Companion` won't be needed